redisgraph/edge.py

```diff
@@ -26,7 +28,10 @@ def toString(self):
 
     def __str__(self):
         # Source node.
-        res = '(' + self.src_node.alias + ')'
+        if isinstance(self.src_node, Node):
+            res = '(' + self.src_node.alias + ')'
```

Comment:
I think we should use Node's `__str__` function in this case.

Reply:
Good idea. Do we have to concern about the query size? Because the commit will call `str(edge)` to construct the create query, and showing both `src` and `dest` nodes could lead to much longer create query.